File: Square.hs
-- Haskell Example 1
-- Purely functional

getSquareArea :: Integer -> Integer     -- function declaration
getSquareArea size = size * size        -- function definition

main = do
    putStrLn("Area of square size 5 has area: ")
    print(getSquareArea 5)              -- prints 25
    
    putStrLn("Area of square size 10 has area: ")
    print(getSquareArea 10)             -- prints 100

File: Square.java
// Java Example 1 
// Object Oriented

public class Square {
    private int size;

    public Square(int size) {
        this.size = size;
    }

    public int getArea() {
        return size * size;
    }

    public static void main(String[] args) {
        Square s = new Square(5);
        System.out.println(s.getArea());    // prints 25
    }
}

